test/e2e/Tests/build-e2e-test.ps1

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env pwsh
+#
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+[CmdletBinding()]
+param(
+    [switch]
+    $Clean,
+
+    [Switch]
+    $SkipStorageEmulator,
+
+    [Switch]
+    $SkipCosmosDBEmulator,
+
+    [Switch]
+    $SkipCoreTools,
+
+    [Switch]
+    $SkipBuildOnPack
+)
+
+$ProjectBaseDirectory = "$PSScriptRoot\..\..\..\"
+$ProjectTemporaryPath = Join-Path ([System.IO.Path]::GetTempPath()) "DurableTaskExtensionE2ETests"
+mkdir $ProjectTemporaryPath -ErrorAction SilentlyContinue > $Null
+$WebJobsExtensionProjectDirectory = Join-Path $ProjectBaseDirectory "src\WebJobs.Extensions.DurableTask"
+$WorkerExtensionProjectDirectory = Join-Path $ProjectBaseDirectory "src\Worker.Extensions.DurableTask"
+$E2EAppProjectDirectory = Join-Path $ProjectBaseDirectory "test\e2e\Apps\BasicDotNetIsolated"
+
+$FunctionsRuntimeVersion = 4
+
+# A function that checks exit codes and fails script if an error is found 
+function StopOnFailedExecution {
+  if ($LastExitCode) 
+  { 
+    exit $LastExitCode 
+  }
+}
+
+$FUNC_CLI_DIRECTORY = Join-Path $ProjectTemporaryPath 'Azure.Functions.Cli'
+if($SkipCoreTool -or (Test-Path $FUNC_CLI_DIRECTORY))
+{
+  Write-Host "---Skipping Core Tools download---"  
+}
+elseif (Test-Path $FUNC_CLI_DIRECTORY) 
```

Comment:
Seems this `elseif` is redundant with the check above.

Reply:
Removed it